docs/source/1.0/spec/core/traits.rst

```diff
@@ -191,6 +191,158 @@ to call ``CreateTable`` on a table that already exists will return an error.
         }
 
 
+.. smithy-trait:: smithy.api#notProperty
+.. _notproperty-trait:
+
+---------------------
+``notProperty`` trait
+---------------------
+
+Summary
+    Used to explicitly mark a top-level input or output shape member of an
+    instance operation is not bound to a resource property. Can also mark
+    another trait to have same effect.
+Trait selector
+    ``:is(operation -[input, output]-> structure > member, [trait|trait])``
+
+    *A top-level member of an operation's input or output shape, or a trait*
+Value type
+    Annotation trait.
+
+By default, top-level input and output members of lifecycle operations must
+be bound to a resource property or identifier. Members that are neither, must
+have this trait applied, or be annotated with a different trait that has this
+trait applied.
+
+.. code-block:: smithy
+
+    resource Forecast {
+        properties: { chanceOfRain: Float }
+        update: UpdateForecast
+    }
+
+    operation UpdateForecast {
+       input: UpdateForecastInput
+    }
+
+    structure UpdateForecastInput {
+        chanceOfRain: Float
+
+        @notProperty
+        dryRun: Boolean
+
+        @idempotencyToken
+        clientToken: String
+    }
+
+
+.. smithy-trait:: smithy.api#property
+.. _property-trait:
+
+------------------
+``property`` trait
+------------------
+
+Summary
+    Binds a top-level input or output shape member of a structure to a resource
+    property with a different name.
+Trait selector
+    ``:is(operation -[input, output]-> structure > member)``
+
+    *A top-level member of an operation's input or output shape*
+Value type
+    An object with the following properties:
+
+    .. list-table::
+       :header-rows: 1
+       :widths: 10 23 67
+
+       * - Property
+         - Type
+         - Description
+       * - name
+         - ``string``
+         - Name of the resource property to bind the member to. This field MUST
+           be specified.
+
+By default, top-level input or output shape members are bound to the resource
+property with the same name. In situations where this isn't possible, this
+trait can be used to specify which property the member is bound to.
+
+   .. admonition:: Note
+    :class: tip
```

Comment:
is this rendering correctly? The indentation seems off.

Reply:
It was off, and now it's rendering correctly.